test/e2e/cypress/integration/users.spec.js

```diff
@@ -1,5 +1,6 @@
 describe('Users', function() {
   const kuzzleUrl = 'http://localhost:7512'
+  const backendVersion = Cypress.env('BACKEND_VERSION') || 2
```

Comment:
Why is there a difference between here and the other `spec.js` files ? 
Also since the only use of `backendVersion` is with `initLocalEnv()` wich set a default value to that parameter it seems useless to do it another time here